clang/unittests/Format/FormatTestComments.cpp

```diff
@@ -796,6 +796,24 @@ TEST_F(FormatTestComments, ParsesCommentsAdjacentToPPDirectives) {
             format("namespace {}\n   /* Test */    #define A"));
 }
 
+
+TEST_F(FormatTestComments, DeIdentsCommentBeforeIfdefAfterBracelessIf) {
+  EXPECT_EQ("void f() {\n"
```

Comment:
Please use `verifyFormat`, I know this file is not a good example, but there are some usages already.

Reply:
Updated to do so. Not sure if I need to give a style.

Reply:
You don't need to because the default style is `LLVM`.